includes/wikia/ExternalUser_Wikia.php

```diff
@@ -204,15 +210,30 @@ public function getLastAuthenticationError() {
 	}
 
 	public function authenticate( $password ) {
+		// Log to check whether this is used anymore (it shouldn't be), so we can clean
+		// this up.
+		// @todo remove after the new password hashing is implemented (PLATFORM-2526)
+		Wikia\Logger\WikiaLogger::instance()->debug(
+			'NEW_HASHING ExternalUser authenticate called',
+			[
+				'user_id' => $userId,
+				'caller' => wfGetCaller(),
+				'exception' => new Exception()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "exception" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)